global_config/development.sh

```diff
@@ -0,0 +1,7 @@
+CONFIG=development
```

Comment:
there's no development environment in the onboarding doc. Just review, test and production. Maybe confirm with the devs.